src/registry_schemas/version.py

```diff
@@ -22,4 +22,4 @@
 Development release segment: .devN
 """
 
-__version__ = '1.2.0'  # pylint: disable=invalid-name
+__version__ = '1.6.2'  # pylint: disable=invalid-name
```

Comment:
Big jump :)

Reply:
The schema has Git releases which this value was not reflecting. The next Git release is 1.6.2.

Reply:
Nice! All caught up